### PR TITLE
Set up routing for the application

### PR DESCRIPTION
**Pull request by GergKllai1**
# PT Link

[Routing](https://www.pivotaltracker.com/story/show/163561124)

# Description of PR

- Sets up react router dom
- Sets up App.jsx as main component with routing
- Passes down props from view to child component using routing

### CODE REVIEW

File: client/bundles/Main/components/App.jsx [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import React from 'react'
+import {Route, Switch, BrowserRouter} from 'react-router-dom'
+import Main from './Main'
+
+
```
Comment (schneiderman805) [inline client/bundles/Main/components/App.jsx:4]: remove empty line

**Review by schneiderman805 — COMMENTED** [review]
just one thing that i can see

**Review by kianaditya — CHANGES_REQUESTED** [review]
How are we testing this?

**Review by HannaSwDn — APPROVED** [review]

File: client/bundles/Main/components/App.jsx (hunk 1 above)
Comment (GergKllai1) [reply client/bundles/Main/components/App.jsx:4]: Done

**Comment by GergKllai1** [comment]
> How are we testing this?

We use acceptance testing to test whether we reach the components we want

**Review by schneiderman805 — APPROVED** [review]

**Review by kianaditya — APPROVED** [review]

**Pull request merged**